rfc8446.xml

```diff
@@ -830,6 +830,9 @@ processed and transmitted as specified by the current active connection state.</
        certificate_request(13),
        certificate_verify(15),
        finished(20),
+       certificate_URL_RESEREVED(21),
```

Comment:
the registry has "url" in lowercase

Reply:
Also, an extra 'E' snuck in to RESER**E**VED.